setup.cfg

```diff
@@ -110,6 +110,7 @@ docs =
   astroquery
   hvpy>=1.0.1
   jplephem
+  spiceypy
```

Comment:
Could we add a new optional dependency category, e.g. `spice` or even `spiceypy`? We already do this with `dask`. It seems weird that a user would need to manage this dependency with the `docs` optional dependency category

Reply:
+1 to `[spice]`.